pkg/controller/eventqueue.go

```diff
@@ -6,40 +6,47 @@
 package controller
 
 import (
-	"github.com/Azure/application-gateway-kubernetes-ingress/pkg/k8scontext"
 	"time"
 
 	"github.com/golang/glog"
 	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/client-go/util/workqueue"
 
+	"github.com/Azure/application-gateway-kubernetes-ingress/pkg/k8scontext"
 	"github.com/Azure/application-gateway-kubernetes-ingress/pkg/utils"
 )
 
+// QueuedEventEventProcessor provides a mechanism to act on events in the internal queue.
+type QueuedEventEventProcessor interface {
```

Comment:
Why not just `EventProcessor`?This is pretty verbose.

Reply:
Oh wow - this indeed got out of hand.